### PR TITLE
WIM-33 - Added News content type and core feature.

### PR DESCRIPTION
**Pull request by ozin7**

### CODE REVIEW

File: drupal-org.make [hunk 1]
```diff
@@ -4,9 +4,13 @@ core = 7.x
 ; Default configuration
 defaults[projects][subdir] = contrib
 
-; Modules
+; Contributed modules.
 projects[jquery_update] = 2.7
```
Comment (T2L) [inline drupal-org.make]: Please list all dependencies in alphabetical order

File: modules/wim_features/wim_news/wim_news.info [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+features[field_instance][] = node-news-field_publication_date
+features[node][] = news
+features[variable][] = menu_options_news
+features[variable][] = menu_parent_news
+features[variable][] = node_options_news
+features[variable][] = node_preview_news
+features[variable][] = node_submitted_news
+features_exclude[field_base][body] = body
+features_exclude[dependencies][date] = date
+project path = profiles/wim/modules/wim_features
```
Comment (T2L) [inline modules/wim_features/wim_news/wim_news.info]: Must be under features directory

File: wim.info [hunk 3]
```diff
@@ -5,3 +5,17 @@ exclusive = TRUE
 core = 7.x
 
 files[] = wim.test
+
+; Install necessary core modules.
+dependencies[] = menu
+dependencies[] = toolbar
```
Comment (T2L) [inline wim.info]: Do we really need toolbar?

File: wim.info [hunk 4]
```diff
@@ -5,3 +5,17 @@ exclusive = TRUE
...
+dependencies[] = image
+
+; Install contributed modules.
+dependencies[] = features
```
Comment (T2L) [inline wim.info]: List in alphabetical order please

File: modules/wim_features/wim_news/wim_news.info (hunk 2 above)
Comment (bramtenhove) [inline modules/wim_features/wim_news/wim_news.info]: like profiles/wim/modules/features?

Comment (T2L) [inline modules/wim_features/wim_news/wim_news.info]: Yes. What do you think?

Comment (bramtenhove) [inline modules/wim_features/wim_news/wim_news.info]: Yeah, sure! Was just wondering

File: modules/wim_features/wim_core/wim_core.strongarm.inc [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'admin_theme';
+  $strongarm->value = 'seven';
+  $export['admin_theme'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'theme_default';
```
Comment (T2L) [inline modules/wim_features/wim_core/wim_core.strongarm.inc]: Having a default theme set in a feature is not the best option. Some municipalities will have a custom theme. Feature will be overridden all the time

Comment (bramtenhove) [inline modules/wim_features/wim_core/wim_core.strongarm.inc]: Let's set the default theme during profile install.

Municipality will then be able to override with their own feature. How about that?

Comment (ozin7) [inline modules/wim_features/wim_core/wim_core.strongarm.inc]: Ok, I will move it to the install file.

**Pull request merged**